### PR TITLE
Add docs about running tests with snowplow

### PR DESCRIPTION
**Pull request by uladzimirdev**
Docs about running snowplow tests

### CODE REVIEW

**Comment by ranquild** [comment]
@uladzimirdev snowplow docs are here https://www.notion.so/metabase/Snowplow-integration-5da1f874beda4153b4fccfa6c1e77caa

**Review by ranquild — APPROVED** [review]

**Comment by uladzimirdev** [comment]
@ranquild ah, I found only [different link in notion](https://www.notion.so/metabase/Snowplow-testing-49ff8a34ba4f42cb996e269b3285f454) and it didn't contain any reference to the doc you mentioned. 

Should we close this PR?

File: docs/developers-guide/e2e-tests.md [hunk 1]
```diff
@@ -84,6 +84,13 @@ A subset of our tests depend on the external services that are available through
 
 Please note the minus sign before the `@external` tag. For more details, consult [the official documentation](https://github.com/cypress-io/cypress-grep#filter-with-tags).
 
+### Running tests with Snowplow involved
+
+Some tests depend on Snowplow and expect a running locally server. To run those tests, you need to:
```
Comment (nemanjaglumac) [inline docs/developers-guide/e2e-tests.md:89]: Nit

```suggestion
Tests that depend on Snowplow expect a server running locally. To run them, you need to:
```
or

```suggestion
Tests that depend on Snowplow expect a locally running server. To run them, you need to:
```

or probably the most terse

```suggestion
Tests that depend on Snowplow expect a running server. To run them, you need to:
```

**Comment by nemanjaglumac** [comment]
> Should we close this PR?

I think it makes sense to have a public facing documentation that lives in GitHub. Do not close it.

**Pull request merged**